src/components/views/dialogs/ChatInviteDialog.js

```diff
@@ -155,14 +155,36 @@ module.exports = React.createClass({
     },
 
     onQueryChanged: function(ev) {
-        var query = ev.target.value;
-        var queryList = [];
+        const query = ev.target.value;
+        let queryList = [];
 
         // Only do search if there is something to search
-        if (query.length > 0) {
+        if (query.length > 0 && query != '@') {
+            // filter the known users list
             queryList = this._userList.filter((user) => {
                 return this._matches(query, user);
+            }).map((user) => {
+                return {
```

Comment:
it'd be helpful to record the structure of the objects in `queryList` and `inviteList` somewhere (in `getInitialState`?)